### PR TITLE
[dom-gpu] update ParaView for 20.08

### PR DESCRIPTION
**Pull request by jfavre**
== COMPLETED: Installation ended successfully (took 21 min 57 sec)
== Results of the build can be found in the log file(s) /scratch/snx3000tds/jfavre/daint/software/ParaView/5.8.1-CrayGNU-20.08-EGL-python3/easybuild/easybuild-ParaView-5.8.1-20200812.152041.log

### CODE REVIEW

**Comment by jfavre** [comment]
addresses https://jira.cscs.ch/browse/UES-1029

File: easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-CrayGNU-20.08-EGL-python3.eb [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+toolchain = {'name': 'CrayGNU', 'version': '20.08'}
+toolchainopts = {'pic': True, 'usempi': True, 'verbose': False}
+
+local_download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % local_download_suffix]
+sources = ["ParaView-v%(version)s.tar.gz"]
+
+dependencies = [
+#    ('h5py', '2.8.0', '-python3-serial'),
```
Comment (lucamar) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-CrayGNU-20.08-EGL-python3.eb:20]:
```suggestion
```

Comment (lucamar) [reply easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-CrayGNU-20.08-EGL-python3.eb:20]: Thanks @jfavre: if the `h5py` dependency is not needed any longer, I suggest to remove it from the list

**Comment by jfavre** [comment]
I am getting a non-fast-forward rejection, so I prefer to wait for the second recipe (the one with Mesa) and I will update the production list at once. Is that ok with you @lucamar ?

**Comment by lucamar** [comment]
> I am getting a non-fast-forward rejection, so I prefer to wait for the second recipe (the one with Mesa) and I will update the production list at once. Is that ok with you @lucamar ?
That's fine for me, however the recipe with `Mesa` goes in the other production list `dom-mc`

File: easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-CrayGNU-20.08-EGL-python3.eb (hunk 1 above)
Comment (jfavre) [reply easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-CrayGNU-20.08-EGL-python3.eb:20]: I have temporarily removed the h5py in both ParaView recipes in order to move forward with testing the app. I plan to add it at a later time. It is an optional component.

**Comment by teojgo** [comment]
@jfavre Can you try with these h5py versions: https://github.com/eth-cscs/production/pull/1855

**Pull request merged**